elasticluster/share/playbooks/roles/slurm-resume.yml

```
@@ -0,0 +1,46 @@
+---
+
+- name: Restart SLURMcltd
+  hosts: slurm_master
+  tasks:
+    - service:
+        name=slurmctld
```

Please prefer the YAML syntax to the old Ansible syntax (here and in the following lines).
```suggestion
        name: slurmctld
```

(I know it's still used in some playbooks... takes time to update all legacy code...)